### PR TITLE
Unnecessary mandatory WCS 1.1.1 parameters

### LINKED ISSUE #306: Unnecessary mandatory parameters on WCS 1.1.1 GetCoverage request
While trying to import the WCS endpoint in GIS software such as ESRI ArcMap or FME, those tools will execute GetCapabilities and DescribeCoverage queries succesfully but will fail to perform the GetCoverage request because GeoTrellis-Server will raise a BadBrequest error.

Here is an example of a WCS GetCoverage request as performed by ESRI ArcMap
https://datacube.services.geo.ca/ows/elevation?SERVICE=WCS&VERSION=1.1.1&REQUEST=GetCoverage&IDENTIFIER=hrdsm&FORMAT=image/geotiff&BOUNDINGBOX=-2282412,3898486,-2282410,3898488,urn:ogc:def:crs:EPSG::3979&RangeSubset=contents:nearest

This request is considered incomplete because of missing parameters and will return the following error message:
`Error parsing parameters: Missing parameter 'gridbasecrs'; Missing parameter 'gridcs'; Missing parameter 'gridtype'; Missing parameter 'gridorigin'; Missing parameter 'gridoffsets'`

According to the list of GetCoverage request parameters (Table 27) in WCS 1.1.1 specifications (https://portal.opengeospatial.org/files/07-067r2), parameters `gridbasecrs`, `gridcs`, `gridtype`, `gridorigin` and `gridoffsets` are a group of optional parameters.

The expected behavior would be that WCS GetCoverage request with only mandatory parameters should be supported.

### PR DESCRIPTION
**Pull request by pomadchin**
## Overview

This PR alignes query params with the WCS 1.1.1 specs.

### Checklist

- [x] Description of PR is in an appropriate section of the CHANGELOG and grouped with similar changes if possible

Closes #306

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -12,6 +12,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## Added
 - WCS services support configuration of `supported-projections` [#314](https://github.com/geotrellis/geotrellis-server/pull/314)
 - Add WCS 1.1.0 as the supported version [#330](https://github.com/geotrellis/geotrellis-server/pull/330)
+- Unnecessary mandatory parameters on WCS 1.1.1 GetCoverage request [#306](https://github.com/geotrellis/geotrellis-server/issues/306)
```
Comment (jisantuc) [inline CHANGELOG.md:15]: This more of a `changed` yeah?

**Review by jisantuc — COMMENTED** [review]
The linked issue says `gridOffsets` is also optional, but it didn't get the `validatedOptionalParam` treatment here -- is that deliberate or is it not necessary to make that one optional?

**Comment by pomadchin** [comment]
@jisantuc I just forgot about it, thanks!

**Review by jisantuc — APPROVED** [review]
😎

**Pull request merged**